releasenotes/notes/no-sympy-19687d5ca814042c.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    ``sympy`` is no longer a requirement for installing Qiskit. After the migration to
+    a Rust based symbolic engine for the :class:`.ParameterExpression` class the uses
+    of Sympy are isolated to some visualization utilities and the
```

Comment:
`ParameterExpression.sympify` also uses it, right? And pending on the other comment on the parameter test maybe some other places also still rely on it 🤔